include/materials/mohr_coulomb.tcc

```diff
@@ -112,6 +114,8 @@ bool mpm::MohrCoulomb<Tdim>::compute_stress_invariants(
                         6.0 +
                     std::pow(stress(3), 2) + std::pow(stress(4), 2) +
                     std::pow(stress(5), 2);
+  // Compute equivalent (Von-Mises) stress
+  (*state_vars).at("evms") = std::sqrt(3 * j2);
```

Comment:
It's seems the same with "q", which may be used more often as the name

Reply:
I am open to the naming. But can we make it in the next PR where we clean up the entire existing laws?

Reply:
I know, but I just use the same parameter "evms" as "q" in MCC.

Reply:
I think the `state_vars` should only contain state variables that need to be passed to the next time step. `p` and `q` or von mises stresses can be computed, and thus should not be part of the `state_vars`

Reply:
Agree with @ezrayst Since we don't need Von-Mises stresses to be calculated, I don't think it's good to have them stored in `state_vars`. If you want to plot, you can use the calculator function in Paraview or HDF5 to compute these.